msteams-platform/tabs/design/tabs.md

```diff
@@ -138,6 +138,8 @@ Notify users of tab activity one of the following ways:
 
 ## Best practices
 
+These recommendations can help ensure a high-quality app experience. While there may be some overlap, you should also make sure your app adheres to the [Teams store validation guidelines](~/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md).
```

Comment:
@ramendhe, trying to make it clearer that design best practices have a different purpose than validation guidelines. Does this work? We can add to each design page, too.